.github/workflows/core.yml

```
@@ -65,6 +65,24 @@ jobs:
     - name: Run tests
       timeout-minutes: 15
       continue-on-error: False
+      shell: bash
       run: |
+        flags=()
+        source=${{ github.event.before }}
+        target=${{ github.event.after }}
+        # Check if a networking file is involved
+        patterns="\
+        ^yt_dlp/networking/
+        ^yt_dlp/utils/networking.py
+        ^test/test_networking.py$
+        ^test/test_networking_utils.py$
+        ^test/test_websockets.py$
+        ^test/test_socks.py$
+        ^pyproject.toml$
+        "
+        if git diff --name-only "${source}" "${target}" | grep -Ef <(printf '%s' "${patterns}"); then
+          flags+=(--flaky)
+        fi
+
         python3 -m yt_dlp -v || true  # Print debug head
```

can we remove this newline? my editor doesn't like empty lines in yml text blocks
```suggestion
        python3 -m yt_dlp -v || true  # Print debug head
```

---

Your editor is sad :'(